server/cli/launch.py

```diff
@@ -42,7 +42,7 @@ def annotation_args(func):
         "Incompatible with --annotations-output-dir.",
     )
     @click.option(
-        "--experimental-annotations-output-dir",
+        "--annotations-output-dir",
```

Comment:
Similar to above, given that we also _load_ data from this directory, I suggest it be renamed `--annotations-directory`

CC: @liaprins-czi

Reply:
That makes sense; I agree with it being `--annotations-directory` for that reason

Reply:
Kk, will update